lib/VerifiableCredentialStore.js

```diff
@@ -113,6 +113,38 @@ export default class VerifiableCredentialStore {
     return doc.content;
   }
 
+  /**
+   * Updates a verifiable credential in remote private storage
+   *
+   * @param {object} credential
+   * @param {object} meta
+   */
+  async update({credential, meta = {}}) {
+    try {
+      const {invocationSigner} = this;
+      const {documents: [doc]} = await this.edv.find({
+        equals: {'content.id': credential.id},
+        invocationSigner
+      });
+      if(!doc) {
+        return false;
+      }
+      if(meta) {
+        doc.meta = meta;
+      }
+      doc.content = credential;
+      return this.edv.update({
+        doc,
+        invocationSigner
+      });
```

Comment:
If you would like an error throw by `edv.update` to be handled by the `catch` on L140, you must `await` the API call in this scope.

https://replit.com/@mattcollier/OrangePolishedInfinity#index.js